src/templates/next/components/internal/Footer/Footer.tsx

```diff
@@ -183,49 +183,51 @@ const LegalSection = ({
   | "siteMapLink"
 >) => {
   return (
-    <div className="flex flex-col gap-4 lg:gap-2">
-      <p className="text-caption-01 text-content-inverse-light">
-        &copy; {new Date().getFullYear()}{" "}
-        {isGovernment ? "Government of Singapore" : agencyName}, last updated{" "}
-        {lastUpdated}
-      </p>
-      <div className="text-caption-01 flex flex-col gap-3 lg:flex-row lg:gap-8">
-        {isGovernment && (
-          <FooterItem
-            title="Report Vulnerability"
-            url="https://tech.gov.sg/report_vulnerability"
-            LinkComponent={LinkComponent}
-          />
-        )}
-        {privacyStatementLink && (
-          <FooterItem
-            title="Privacy Statement"
-            url={privacyStatementLink}
-            LinkComponent={LinkComponent}
-          />
-        )}
-        {termsOfUseLink && (
-          <FooterItem
-            title="Terms of Use"
-            url={termsOfUseLink}
-            LinkComponent={LinkComponent}
-          />
-        )}
-        {isGovernment && (
-          <FooterItem
-            title="Reach"
-            url={"https://www.reach.gov.sg"}
-            LinkComponent={LinkComponent}
-          />
-        )}
+    <div className="flex h-full">
+      <div className="flex flex-col	justify-end	gap-4 lg:gap-2">
```

Comment:
nit: feels like some extra spaces between the classes here

Reply:
fixed in [935cead](https://github.com/opengovsg/isomer-next/pull/108/commits/935cead4c37ef853bd0f16b3d390ee928f7632d3)!